contracts/upgrade/utils/OmnibusBase.sol

```diff
@@ -41,41 +45,66 @@ abstract contract OmnibusBase {
         ScriptCall call;
     }
 
-    IVoting private immutable VOTING_CONTRACT;
+    IVoting internal immutable VOTING_CONTRACT;
+    IDualGovernance internal immutable DUAL_GOVERNANCE;
 
-    constructor(address voting) {
+    constructor(address voting, address dualGovernance) {
         VOTING_CONTRACT = IVoting(voting);
+        DUAL_GOVERNANCE = IDualGovernance(dualGovernance);
     }
 
     /// @return VoteItem[] The list of voting items to be executed by Aragon Voting.
     function getVoteItems() public view virtual returns (VoteItem[] memory);
 
+    function getVotingVoteItems() public view virtual returns (VoteItem[] memory votingVoteItems) {
+        uint256 numVotingVoteItems = 0;
+        votingVoteItems = new VoteItem[](numVotingVoteItems);
+        uint256 index = 0;
+
+        assert(index == numVotingVoteItems);
```

Comment:
shall we make it throwing (just make abstract)

Reply:
done